lib/beaker/host/windows/exec.rb

```diff
@@ -5,7 +5,7 @@ def reboot
     exec(Beaker::Command.new('shutdown /r /t 0 /d p:4:1 /c "Beaker::Host reboot command issued"'), :expect_connection_failure => true)
   end
 
-  ABS_CMD = 'c:\\\\windows\\\\system32\\\\cmd.exe'
+  ABS_CMD = "/cygdrive/c/WINDOWS/system32/cmd.exe"
```

Comment:
I think it's more that in the acceptance test, you could call the `Host#echo` method, and get a string back that could then be used in a manifest, eg https://github.com/puppetlabs/puppet/blob/master/acceptance/tests/language/resource_refs_with_nested_arrays.rb#L7